runtime/common/src/evm/precompile.rs

```diff
@@ -94,11 +99,75 @@ where
 	}
 }
 
+<<<<<<< HEAD
+/// A set of precompiles. This set might contain
+/// not yet mainnet ready precompiles in order to test
+/// those in development or staging environment without touching
+/// the mainnet set.
+pub struct Development<R>(PhantomData<R>);
+
+impl<R> Development<R> {
+	#[allow(clippy::new_without_default)] // We'll never use Default and can't derive it.
+	pub fn new() -> Self {
+		Self(Default::default())
+	}
+}
+
+impl<R> PrecompileSet for Development<R>
+where
+	R: pallet_evm::Config + axelar_gateway_precompile::Config + frame_system::Config,
+	R::RuntimeCall: Dispatchable<PostInfo = PostDispatchInfo> + GetDispatchInfo + Decode,
+	<R::RuntimeCall as Dispatchable>::RuntimeOrigin: From<Option<R::AccountId>>,
+	axelar_gateway_precompile::Pallet<R>: Precompile,
+{
+	fn execute(&self, handle: &mut impl PrecompileHandle) -> Option<PrecompileResult> {
+		match handle.code_address().0 {
```

Comment:
...

Which would allow writing this as:
```
self.0.execute(handle).or_else(|| {
  match handle.code_address().0 {
    CONNECTORS_AXELAR_GATEWAY => ...,
    // Additional development-specific ones
    _ => None
  }
})
```

Reply:
Good idea. Will do that!